app/utils/helpers.ts

```diff
@@ -182,3 +182,12 @@ export function areBothStringArraysEqual(a: string[], b: string[]) {
 
     return areBothEqual;
 }
+
+export function isValidUrl(url: string): boolean {
```

Comment:
there is an `isValidUrl` function in `app/utils/url` which may cause some confusion to devs, either be more specific, use or modify the one we already have. Also if you decide to keep this and change the name, make sure to at least move it the right file, and do add test coverage for it